jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/ManagementUtils.java

```diff
@@ -770,6 +774,16 @@ static void registerAll() {
 				.addInterface(com.ibm.lang.management.JvmCpuMonitorMXBean.class)
 				.validateAndRegister();
 
+			create(OPENJ9_DIAGNOSTICS_MXBEAN_NAME, openj9.lang.management.internal.OpenJ9DiagnosticsMXBeanImpl.getInstance())
+				.addInterface(openj9.lang.management.OpenJ9DiagnosticsMXBean.class)
+				.validateAndRegister();
+
+			/*[IF JAVA_SPEC_VERSION >= 21]*/
+			create(HOTSPOT_DIAGNOSTICS_MXBEAN_NAME, com.sun.management.internal.ExtendedHotSpotDiagnostic.getInstance())
+				.addInterface(com.sun.management.HotSpotDiagnosticMXBean.class)
+				.validateAndRegister();
+			/*[ENDIF] JAVA_SPEC_VERSION >= 21 */
```

Comment:
This, and the declaration of `HOTSPOT_DIAGNOSTICS_MXBEAN_NAME` above constitude dead code as it falls between line 604
```java
	/*[IF JAVA_SPEC_VERSION == 8]*/
```
and line 1042
```java
	/*[ENDIF] JAVA_SPEC_VERSION == 8 */
```

Reply:
Right, my bad since I mentioned adding in the Hotspot bean, forgetting it was only added for jdk21+.

Reply:
Removed the `HOTSPOT_DIAGNOSTICS_MXBEAN_NAME` code since it is not needed.